docs/docs/ProgrammingGuide/run-on-k8s.md

```diff
@@ -49,7 +49,7 @@ $SPARK_HOME/bin/spark-submit \
   --conf spark.kubernetes.executor.docker.image=intelanalytics/spark-executor:v2.2.0-kubernetes-0.5.0-ubuntu-14.04 \
   --conf spark.kubernetes.initcontainer.docker.image=intelanalytics/spark-init:v2.2.0-kubernetes-0.5.0-ubuntu-14.04 \
   --conf spark.kubernetes.resourceStagingServer.uri=http://<address-of-any-cluster-node>:31000 \
-  $BIGDL_HOME/lib/bigdl-0.4.0-SNAPSHOT-jar-with-dependencies.jar \
+  $BIGDL_HOME/lib/bigdl-0.6.0-SNAPSHOT-jar-with-dependencies.jar \
```

Comment:
why 0.6.0-SNAPSHOT? The code version is 0.7.0-SNAPSHOT, so should it be 0.7.0-SNAPSHOT? And when release it should be changed to 0.7.0, right?

Reply:
exactly! The part will be changed.